src/components/Locators/Locator.jsx

```diff
@@ -48,6 +54,8 @@ export const Locator = memo(({ element, currentPage, noScrolling }) => {
 
   const ref = useRef(null);
 
+  const isLocatorInProgress = Object.values(locatorProgressStatus).includes(element.locator.taskStatus);
```

Comment:
that's correct, but, please, have a look at `isProgressStatus` in src\services\locatorGenerationController.js, it does the same. If you'll find a more predictable place for this function, please, feel free and move it )